gcloud-java-pubsub/baseline/src/main/java/com/google/cloud/pubsub/spi/v1/PublisherApi.java

```diff
@@ -289,15 +297,26 @@ public void close() throws IOException {
    * @throws com.google.api.gax.grpc.ApiException if the remote call fails
    */
   public final Topic createTopic(String name) {
+    TOPIC_PATH_TEMPLATE.validate(name);
     Topic request = Topic.newBuilder().setName(name).build();
-
     return createTopic(request);
   }
 
   // AUTO-GENERATED DOCUMENTATION AND METHOD - see instructions at the top of the file for editing.
   /**
    * Creates the given topic with the given name.
    *
+   * Sample code:
+   * <pre><code>
+   * try (PublisherApi publisherApi = PublisherApi.createWithDefaults()) {
+   *   String formattedName = PublisherApi.formatTopicName("[PROJECT]", "[TOPIC]");
```

Comment:
Weird question -- can createWithDefaults pick up the project ID from the environment ? @garrettjonesgoogle